libs/data_providers_sdk/src/price_data/fetchers/stock_markets/utils.rs

```diff
@@ -0,0 +1,18 @@
+pub fn print_missing_network_price_data<T>(
+    network: &str,
+    symbol: String,
+    price: Option<T>,
+    volume: Option<T>,
+) {
+    eprintln!(
+        "[{network}] Skipping symbol {}: missing {}{}{}",
+        symbol,
+        if price.is_none() { "price" } else { "" },
+        if price.is_none() && volume.is_none() {
+            " and "
+        } else {
+            ""
+        },
+        if volume.is_none() { "volume" } else { "" },
+    );
```

Comment:
Something like this may be better for clarity

```suggestion
    let mut missing_fields = Vec::new();

    if price.is_none() {
        missing_fields.push("price");
    }
    if volume.is_none() {
        missing_fields.push("volume");
    }

	// Do we actually want this (the printing) to happen all the time?
    if !missing_fields.is_empty() {
        eprintln!(
            "[{network}] Skipping symbol {symbol}: missing {}",
            missing_fields.join(" and ")
        );
    }
```

Is this function called only when there **is** an error for sure (i.e. there's always at least one `None` in `price` and `volume`)?

Reply:
Any of them or both can be missing

Reply:
With both of them missing we should see `Skipping symbol X: price and volume`